### ISSUE TITLE
Android Plugin 7.0.0 breaking changes

### ISSUE THREAD

**Issue by redwolf2** [Scandit/barcodescanner-sdk-cordova#125]
https://cordova.apache.org/announcements/2017/12/04/cordova-android-7.0.0.html

The manifests sub dir changed. 

Only cordova-android 6.4.0 is working ATM

```
Installing "com.mirasense.scanditsdk.plugin" for android

Failed to install 'com.mirasense.scanditsdk.plugin': Error: ENOENT: no such file or directory, open 'C:\Daten\Projekte\MyProject\platforms\android\AndroidManifest.xml'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.readFileSync (fs.js:551:33)
    at Object.parseElementtreeSync (C:\Daten\Projekte\MyProject\platforms\android\cordova\node_modules\cordova-common\src\util\xml-helpers.js:180:27)
    at new AndroidManifest (C:\Daten\Projekte\MyProject\platforms\android\cordova\lib\AndroidManifest.js:29:20)
    at AndroidProject.getPackageName (C:\Daten\Projekte\MyProject\platforms\android\cordova\lib\AndroidProject.js:99:12)
    at Api.addPlugin (C:\Daten\Projekte\MyProject\platforms\android\cordova\Api.js:223:57)
    at handleInstall (C:\Users\redwolf2\AppData\Roaming\npm\node_modules\cordova\node_modules\cordova-lib\src\plugman\install.js:594:10)
    at C:\Users\redwolf2\AppData\Roaming\npm\node_modules\cordova\node_modules\cordova-lib\src\plugman\install.js:357:28
    at _fulfilled (C:\Users\redwolf2\AppData\Roaming\npm\node_modules\cordova\node_modules\cordova-lib\node_modules\q\q.js:787:54)
    at self.promiseDispatch.done (C:\Users\redwolf2\AppData\Roaming\npm\node_modules\cordova\node_modules\cordova-lib\node_modules\q\q.js:816:30)

(node:8560) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: ENOENT: no such file or directory, open 'C:\Daten\Projekte\MyProject\platforms\android\AndroidManifest.xml'
(node:8560) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

BTW it would be nice to have npm integration.

**Comment by redwolf2** [comment]
Fixed in 5.6?
https://ssl.scandit.com/sdk/release-notes/scandit-barcodescanner-linux_5.6.0

**Comment by lucatorella** [comment]
Yes, it should be :)
